### DISCUSSION TITLE
Getting error when importing GCP services

### DISCUSSION THREAD

**Discussion by musyaf11** [GoogleCloudPlatform/terraformer#1698] [Q&A] [answered]
Hey Folks, 

FYi, I'm pretty new to Terraformer and am currently experimenting the importing GCP infrastructure. I'm getting errors when run a command to import cloudsql as follows: 

> 047+0700 [ERROR] plugin.terraform-provider-google_v4.63.1_x5: Response contains error diagnostic: tf_provider_addr=registry.terraform.io/hashicorp/google tf_req_id=c4fd8905-5112-3b1b-9cb8-c51c4f0ae679 tf_rpc=Configure @module=sdk.proto
>   diagnostic_detail=
>   | An unexpected error was encountered trying to build a value. This is always an error in the provider. Please report the following to the provider developer:
>   | 
>   | Received null value, however the target type cannot handle null values. Use the corresponding `types` package type, a pointer type or a custom type that handles null values.
>   | 
>   | Path: 
>   | Target Type: google.ProviderModel
>   | Suggested `types` Type: basetypes.ObjectValue
>   | Suggested Pointer Type: *google.ProviderModel
>    diagnostic_severity=ERROR diagnostic_summary="Value Conversion Error" tf_proto_version=5.3 @caller=github.com/hashicorp/terraform-plugin-go@v0.14.3/tfprotov5/internal/diag/diagnostics.go:55 diagnostic_attribute="" timestamp="2023-04-28T17:07:16.047+0700"

What will be the cause here? really appreciated any of help here 🙏 :man_bowing

**Comment by musyaf11** [comment, marked as answer]
I found the way out by downgrading the GCP provider version to v.4.60.0

**Reply by pataquets** [reply]
See also related issue #1695
